src-backbone/app/js/models/procedure.js

```diff
@@ -12,10 +12,12 @@ let Procedure = Backbone.Model.extend({
     urlRoot: '/api/procedures',
 
     defaults: function() {
+        let user = App().session.getUser();
```

Comment:
I added that function to session.js. I can do it that way, but I wanted to abstract away the key-value store.